website/faq.html

```
@@ -218,7 +218,18 @@ <h2>General</h2>
                 </button>
                 <div class="faq-answer">
                     <div class="faq-answer-inner">
-                        Currently, SecureCall is Android-only. An iOS version is planned but not yet under development. The Rust crypto engine is cross-platform, so the core security will be identical.
+                        Currently, SecureCall is Android-only. An iOS version is planned but not yet under development. The Rust crypto engine is cross-platform, so the core security will be identical. The planned iOS client will support external device VPNs but will not provide its own VPN tunnel.
+                    </div>
+                </div>
+            </div>
+            <div class="faq-item">
+                <button class="faq-question">
```

HTMLHint (1.9.2)

[warning] The type attribute must be present on <button> elements.

(button-type-require)